ui/raidboss/data/05-shb/raid/e2n.js

```diff
@@ -295,23 +223,21 @@
       },
     },
     {
-      id: 'E2N Countdown Marker Shadoweye',
+      id: 'E2N Countdown Marker Shadoweye LookAwayFrom',
```

Comment:
Can you call these `E2N Countdown Marker Shadoweye` and `E2N Countdown Marker Shadoweye You` to be consistent with other triggers?